docs/main/02-intro.md

```diff
@@ -1,71 +1,131 @@
+
+Some teams describe Kratix as a platform orchestrator, designed specifically to
+help platform teams deliver more value, faster, safer, and with less overhead.
 
-Kratix is designed and built by a team with decades of experience creating [internal developer platforms](https://www.syntasso.io/internal-developer-platforms). We've learned that great platforms address three key challenges faced by enterprise organisations: 
+## Why Build Your Platform with Kratix?
 
-- **Slow delivery of value** because software engineers must wait weeks for other teams to process their tickets.
-- **Inconsistent application of key business rules** as teams create shadow IT systems to bypass lengthy ticket queues, introducing business risk.
-- **Tech sprawl leading to operational overhead** as SRE, DevOps, and Platform Teams spend their time maintaining services and infrastructure instead of improving them.
+Kratix addresses critical challenges faced by platform teams, providing clear
+benefits:
 
-A [Kratix Promise](./03-reference/11-promises/01-intro.md) enables you to provide anything-as-a-service, eliminating the need for engineers to wait for manual ticket processing.
+### 🚀 Go Faster
 
-[Workflows](./03-reference/12-workflows.md) let you embed any business rule into your Promise definition, from security scans and billing checks to manual approvals.
+Developers lose productivity when forced to wait on manual processes or tickets.
+Kratix lets you offer everything as-a-service via clear APIs, which can easily
+be exposed through developer portals like Backstage. This enables developers to
+instantly self-service resources, significantly accelerating software delivery.
 
-And with Kratix's built-in [Fleet Management](./03-reference/11-promises/03-updates.md) capabilities, you can maintain and upgrade all your resources with a single click.
+### 🔒 Be Safer
 
-![Kratix Architecture](/img/kratix-architecture.svg)
+Inconsistent application of critical processes creates risks. Kratix helps you
+embed essential business rules—such as security, compliance, and
+governance—directly into automated workflows, ensuring reliable enforcement
+every time.
 
-## Who is Kratix for
+### ⚙️ Increase Efficiency
 
-### Platform Engineers
-Teams responsible for building and maintaining platforms often struggle to meet the growing expectations of application teams and organisations. Delivering the right services quickly, safely, and consistently is already challenging—adding Day 2 operations like upgrades and continuous improvements can lead to burnout.
+Manual management of fragmented infrastructure results in operational overhead.
+Kratix simplifies operations by managing platform resources as a unified,
+automated fleet, reducing complexity and scaling effortlessly as your
+organisation grows.
+
+:::info
+
+Want to chat to us about how Kratix can help your organisation? [Contact
+us by clicking the "Book a Demo" button on the Syntasso
+website](https://syntasso.io). 
```

Comment:
```suggestion
Want to get started faster with ready built portal and infrastructure as code integrations and maintain top tier security with secure base images and SBOMs? [Contact
Syntasso about the enterprise offering by clicking the "Book a Demo" button on the Syntasso
website](https://syntasso.io). 
```

Or something about why over and above these points SKE is better...

Reply:
agreed, commiting as a placeholder for now